__tests__/__integration__/cmd/__tests__/integration/cli/gen-help/__snapshots__/Cmd.cli.gen-help.allowable-options.integration.test.ts.snap

```diff
@@ -67,6 +67,10 @@ exports[`cmd-cli gen-help allowable-options should display the help with allowab
  GLOBAL OPTIONS
  --------------
 
+   --show-inputs-only (boolean)
+
+      Show command inputs and do not run the command
+
    --response-format-json | --rfj (boolean)
 
       Produce JSON formatted data from a command
```

Comment:
Would it be ideal for the "Global Options" to be listed in alphabetical order? I know they already were out of order with `--rfj` coming before `-h`, but now that we're adding more options to the list it strikes me as odd that they have no clear order.

Reply:
I think alphabetical menu is a good idea if universally applied.  `ZOSMF CONNECTION OPTIONS`, `RESPONSE FORMAT OPTIONS`, and `OPTIONS` are also non-alphabetical.  

`CommandPreparer.appendAutoOptions` occurs in logic after the new `CommandPreparer.appendPassOnOptions`.  Perhaps this could be adjusted, but I'm unsure of the implications.  `passOn` has a `merge` behavior and changing the ordering might be non-trivial.  Is it acceptable to address menu organization as a separated issue?

Reply:
Good points, changing the order could be non-trivial and seems outside the scope of this PR. Created #761 🙂